### PR TITLE
[CyberDashboard - 88] - List of users as react component

### PR DESCRIPTION
**Pull request by sonishc**
It resolves https://github.com/CyberCraftInc/cyber-dashboard/issues/88

Screenshot

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -26,7 +26,6 @@
 
 Capybara.register_driver :selenium do |app|
   Capybara::Selenium::Driver.new(app, browser: :chrome)
-  Capybara::Selenium::Driver.new(app, browser: :firefox)
```
Comment (vkostyrka) [inline spec/rails_helper.rb]: I use firefox, please leave this config

**Comment by sonishc** [comment]
> It resolves #88

PTAL @alazarchuk and @rubycop

File: app/javascript/components/Users.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        listOfUsers.first_name +
+        listOfUsers.last_name +
+        listOfUsers.position
+      )
+        .toLowerCase()
+        .includes(search.toLowerCase())
+    );
+
+  sortBy = type => {
+    type === "position";
```
Comment (alazarchuk) [inline app/javascript/components/Users.js]: Is it necessary?

Comment (sonishc) [reply app/javascript/components/Users.js]: @alazarchuk equal by type? or sort by position?

**Pull request merged**